config/locales/de.yml

```diff
@@ -352,15 +352,20 @@ de:
       title: My jobs
       preview: Preview
       index:
-        description: Du hast %{count} Jobs eingestellt
-        none: Du hast noch keine Jobs eingestellt
+        description: Our aim is to be the one stop shop for junior developer roles. All jobs featured on our job board must fall under the following three categories; paid internships, apprenticeships or junior developer roles. Additionally, jobs must not have a previous experience or degree requirement and must be paid positions. Each job that is submitted will be approved by the codebar team before appearing on the job board.
```

Comment:
Small typo here - should read `have any previous`

Reply:
@KimberleyCook so this was actually written this way on purpose, but it's clearly not clear enough so I'll rephrase.

Reply:
Yeah, it doesn't make much sense 👍

Reply:
@KimberleyCook I updated the copy, let me know if it's clearer.